### PR TITLE
chore(connect-web): make type-check do something

### PR DESCRIPTION
**Pull request by mroz22**

### CODE REVIEW

File: packages/connect-web/tsconfig.json [hunk 1]
```diff
@@ -10,13 +10,6 @@
         ],
         "rootDir": ".."
     },
-    "include": [
-        "./connect-web",
-        "../connect-iframe/webpack/prod.webpack.config.ts",
-        "../connect-iframe/package.json",
-        "../connect-popup/webpack/prod.webpack.config.ts",
-        "../connect-popup/package.json"
-    ],
```
Comment (mroz22) [inline packages/connect-web/tsconfig.json]: this include statement was actually causing 
`yarn workspace @trezor/connect-web type-check` to do nothing. I am not sure how to fix this properly. Is there some typescript mage around?

File: packages/connect-popup/webpack/prod.webpack.config.ts [hunk 2]
```diff
@@ -4,6 +4,9 @@ import HtmlWebpackPlugin from 'html-webpack-plugin';
 import TerserPlugin from 'terser-webpack-plugin';
 import CopyPlugin from 'copy-webpack-plugin';
 import * as URLS from '@trezor/urls';
+// todo: https://github.com/trezor/trezor-suite/issues/5305
+/* eslint-disable @typescript-eslint/prefer-ts-expect-error */
+// @ts-ignore
```
Comment (mroz22) [inline packages/connect-popup/webpack/prod.webpack.config.ts]: note that we are already using this pattern here https://github.com/trezor/trezor-suite/blob/develop/packages/connect/e2e/__fixtures__/cardanoGetAddressDerivations.ts#L3

File: packages/connect-web/tsconfig.json (hunk 1 above)
Comment (szymonlesisz) [reply packages/connect-web/tsconfig.json]: "./connect-web" > "." should do the trick
 
 there is no "connect-web" directory inside "packages/connect-web" :)

Comment (mroz22) [reply packages/connect-web/tsconfig.json]: forced push, lets see

**Comment by szymonlesisz** [comment]
after closer inspection turns out that `@suite-common/intl-types` has [the same issue](https://github.com/trezor/trezor-suite/issues/5305#issuecomment-1708574395)

**Review by szymonlesisz — APPROVED** [review]

**Pull request merged**